google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -400,6 +399,30 @@ public void testListBlobsEmptySelectedFields() throws InterruptedException {
       assertTrue(blobSet.contains(remoteBlob.getName()));
       assertNull(remoteBlob.getContentType());
     }
+
+    // Test listing a Requester Pays bucket.
+    Bucket remoteBucket = storage.get(BUCKET, Storage.BucketGetOption.fields(BucketField.ID));
```

Comment:
It seems it should be a separate test, not an addition to the existing one (the original test is already big, this addition makes it huge)

Reply:
Done.

Reply:
Done.